### PR TITLE
LPS-147989 Fix Relationship test failures

### PR DESCRIPTION
**Pull request by thaiscabrals**

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAutoGeneratedLayout.testcase [hunk 1]
```diff
@@ -104,8 +104,6 @@ definition {
 	@description = "LPS-143065 - Verify that the file can be downloaded (a download button appears on the side of the name on the mouse hover)"
 	@priority = "4"
 	test CanDownloadFile {
-		property portal.upstream = "quarantine";
-
 		ObjectAdmin.addObjectViaAPI(
 			labelName = "Custom Object 151581",
 			objectName = "CustomObject151581",
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAutoGeneratedLayout.testcase:109]: Hey @thaiscabrals was this test fixed as well? I didn't see any related changes to this so wasn't sure if this was intentional

**Review by timpak — COMMENTED** [review]
Good job with the fix! Just a couple of things we need to address:
- Please see my question below.
- Please check for tickets to remove the tests from quarantine. If they exist, you can link them to LPS-147989 and close them. Leave a comment stating that the tests were removed from quarantine under LPS-147989.

In the future, please add a link to the ticket in the description. It makes it easier to track what ticket or tickets were worked on. If you need clarification, let me know and I can go over this with you.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAutoGeneratedLayout.testcase (hunk 1 above)
Comment (thaiscabrals) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectAutoGeneratedLayout.testcase:109]: Hi @timpak, I had already linked in LPS-147989, commented and closed.
Next time, I will add the link to the description. Thanks for help.

**Comment by timpak** [comment]
https://github.com/timpak/liferay-portal/pull/283 contains the fix

**Comment by timpak** [comment]
See https://github.com/timpak/liferay-portal/pull/292

**Pull request closed without merging**